arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/Sequence.kt

```diff
@@ -319,6 +360,24 @@ public fun <A, B> Sequence<A>.crosswalk(f: (A) -> Sequence<B>): Sequence<Sequenc
     }
   }
 
+@OverloadResolutionByLambdaReturnType
+@JvmName("crosswalkT")
+public fun <A, B> Sequence<A>.crosswalk(f: (A) -> List<B>): List<List<B>> =
```

Comment:
Could we maybe make this more general?

```suggestion
public fun <A, B> Sequence<A>.crosswalk(f: (A) -> Iterable<B>): List<List<B>> =
```

Reply:
Sure!